common-libs/tiny-lds-ecdsa-secp256k1-2019/package.json

```diff
@@ -9,17 +9,13 @@
     "dist"
   ],
   "scripts": {
-    "start": "tsdx watch --target node --tsconfig ./tsconfig.build.json",
-    "build": "tsdx build --target node --tsconfig ./tsconfig.build.json",
-    "audit": "npm audit",
```

Comment:
Why did you remove this commands? `audit`, `audit:fix`

Reply:
Oops, that was unintenional